build.fsx

```diff
@@ -73,35 +66,20 @@ let vsceTool =
 
 Target "Clean" (fun _ ->
     CleanDir "./temp"
-    CopyFiles "release" ["README.md"; "LICENSE.md"; "RELEASE_NOTES.md"]
```

Comment:
I think I would prefer leaving release `dir` as it is, at least for this first PR. When we get everything working (whole release process) we can think about making repository setup bit better. But let's do small steps now.